lib/crypto/base32.js

```diff
@@ -21,7 +21,7 @@ function base32(len) {
     .then(bytes => {
       const out = []
 
-      for (let i = 0; i < len; i++) {
+      for (let i = 0; i < len; i ++) {
```

Comment:
@divyabiyani these changes need to be part of a new pull request, both in this repo and in the eslint-plugin-fxa repo.

I reopened and cherry picked it here and waiting for the tests to pass: https://github.com/mozilla/fxa-auth-server/pull/1636